server/src/main/java/org/eclipse/lsp/cobol/core/model/extendedapi/CFASTNode.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2021 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.core.model.extendedapi;
+
+import lombok.Data;
+
+import java.util.List;
+
+/**
+ * Data transport object of CF AST.
+ * CFASTNode is root of CF ASF node hierarchy.
+ * It contains list of possible children of CF AST node.
+ */
+@Data
```

Comment:
Why is it `@Data`? Seems like you do not reassign the field.

Reply:
I'm not sure how else to make the class possible to extend. What should I put there instead of ```@Data```?

Reply:
I didn't notice this requirement, now it's ok